contrib/test/integration/system.yml

```diff
@@ -67,3 +67,4 @@
 
 - name: Update the kernel cmdline to include quota support
   command: grubby --update-kernel=ALL --args="rootflags=pquota"
+  when: ansible_distribution == 'RedHat'
```

Comment:
Should this be RHEL and Centos not Red Hat?

Reply:
See https://github.com/mrunalp/cri-o/blob/a18c07f3348941de028bc004ec67c9ef9361066e/contrib/test/integration/system.yml#L12
We use 'RedHat' for RHEL in ansible.

Reply:
Ok I think we should add CenOS to this, so we can handle it there when we start testing on Centos.